code/modules/reagents/chemistry/recipes/drugs.dm

```diff
@@ -47,3 +47,10 @@
 	results = list("happiness" = 4)
 	required_reagents = list("nitrous_oxide" = 2, "epinephrine" = 1, "ethanol" = 1)
 	required_catalysts = list("plasma" = 5)
+	
+/datum/chemical_reaction/synthetic_cocaine
+	name = "Synthetic Cocaine"
+	id = "synthetic_cocaine"
+	results = list("synthetic_cocaine" = 4)
+	required_reagents = list("ephedrine" = 1, "ammonia" = 1, "welding_fuel" = 1, "nitrogen" = 1)
```

Comment:
This is easily made with a single macro+heating. Ignoredamageslowdown is a powerful trait. Would be more interesting if it used one of the lavaland mushroom chems IMO.